### PR TITLE
Code Insights: Remove views area page

### PR DESCRIPTION
**Pull request by vovakulikov**
Closes https://github.com/sourcegraph/sourcegraph/issues/22462

Background

Historically Code Insights pages with insights are built on top of the `ViewContent` component from the` /views` product area. `ViewContent` uses `QueryInputInViewContent` which uses a lot of infrastructure props from the top level of the app. That means that if we want to render code insights on some page we have to specify all these infrastructure props. That requires a lot of information from top-level components and usually inconvenient from a development point of view, plus that increases the complexity of using code insights as independent components.

Changes

This PR removes `/views` area and all related components. By this, all consumers of the `InsightsViewContent` component were supplied and currently don't require a lot of infrastructure props (like pattern type, setting cascade, etc)

### CODE REVIEW

**Review by valerybugakov — APPROVED** [review]
Clean! 🧹  It's so much easier to follow `props` now.

File: client/web/src/insights/components/insight-view-content/chart-view-content/ChartViewContent.scss [hunk 1]
```diff
@@ -1,6 +1,6 @@
-@import './charts/line/LineChart';
-@import './charts/pie/PieChart';
-@import './charts/bar/BarChart';
+@import 'charts/line/LineChart';
+@import 'charts/pie/PieChart';
+@import 'charts/bar/BarChart';
```
Comment (felixfbecker) [inline client/web/src/insights/components/insight-view-content/chart-view-content/ChartViewContent.scss:3]: Why these changes? Seems a bit unconventional, looks more like an import from `node_modules` now

Comment (vovakulikov) [reply client/web/src/insights/components/insight-view-content/chart-view-content/ChartViewContent.scss:3]: @felixfbecker I moved some files (charts) to the insights-related directory and autofix-import has changed these paths. I will return relative paths in the next PRs

File: CHANGELOG.md [hunk 2]
```diff
@@ -31,6 +31,7 @@ All notable changes to Sourcegraph are documented in this file.
 ### Removed
 
 - The experimental paginated search feature (the `stable:` keyword) has been removed, to be replaced with streaming search. [#22428](https://github.com/sourcegraph/sourcegraph/pull/22428)
+- The experimental extensions view page [#22565](https://github.com/sourcegraph/sourcegraph/pull/22565)
```
Comment (felixfbecker) [inline CHANGELOG.md:34]: This should be a complete sentence like the bullet point above :)
